### PR TITLE
Added docs for Zipkin trace propagation (#2071)

### PR DESCRIPTION
**Pull request by peterfroehlich**
Added new propagator with Header information and fixed example.

### CODE REVIEW

File: linkerd/docs/protocol-h2.md [hunk 1]
```diff
@@ -402,13 +402,31 @@ kind: `io.l5d.default`.
 The default trace propagator stores the trace id in the `l5d-ctx-trace` request header.  It also
 reads the `l5d-sample` and, if present, uses this value as the sample rate for this request.
 
-#### Trace Propagator Configuration:
+<aside class="notice">
+The trace information in the header are serialized by Finagles `TraceId.serializeIt` method.
```
Comment (adleong) [inline linkerd/docs/protocol-h2.md]: I can't find the TraceId.serializeIt method.  Am I missing something?

Comment (peterfroehlich) [reply linkerd/docs/protocol-h2.md]: Ah, sorry thats a typo. ^^;;; 
The "Id" part was added by mistake, its TraceId.serialize.

See the Finagle doc: 
https://twitter.github.io/finagle/docs/com/twitter/finagle/tracing/TraceId$.html

**Comment by peterfroehlich** [comment]
Arg, sorry, mixing notebooks and private / professional github accounts made a mess of things. ^^; 
I'll fix it tomorrow at work.

**Comment by muuki88** [comment]
Muki to the rescue 🦄

**Comment by peterfroehlich** [comment]
Superseeded by proper PR #2124

**Pull request closed without merging**